scvelo/tools/_em_model.py

```diff
@@ -158,15 +168,17 @@ def _prepare_genes(self):
             var_names = var_names[np.argsort(np.sum(X, 0))[::-1][: self._n_top_genes]]
         self._var_names = var_names
 
-    # TODO: Implement
     def state_dict(self):
         """Return the state of the model."""
-        raise NotImplementedError
+        return asdict(self._state_dict)
```

Comment:
Can you please undo this implementation? This PR should only concern switching from the current implementation to using `EMParams`. The implementation of these functions is covered in #1025.